SayTheirNames/Source/Components/MainTabBarController.swift

```diff
@@ -64,7 +64,10 @@ class MainTabBarController: UITabBarController, UITabBarControllerDelegate {
         tabBar.barTintColor = defaultBarTint
         tabBar.tintColor = defaultTint
         tabBar.unselectedItemTintColor = defaultUnselectedTint
-        UITabBarItem.appearance().setTitleTextAttributes([NSAttributedString.Key.font: UIFont(name: "Karla-Regular", size: 11)!], for: .normal)
+        guard let karlaFont = UIFont(name: "Karla-Regular", size: 11) else {
```

Comment:
We should try to use our semantic fonts when possible to keep consistency
```suggestion
        let karlaFont = UIFont.STN.tabButtonTitle
```

Reply:
Done: https://github.com/Say-Their-Name/say-their-names-ios/pull/55

Reply:
Done in #55